source/Server/Common/PreCachedAssemblyResolver.cs

```diff
@@ -35,6 +35,8 @@ public class PreCachedAssemblyResolver : ICSharpCode.Decompiler.Metadata.IAssemb
                     return null;
                 if (reference.Name == "System.Threading.AccessControl")
                     return null;
+                if (reference.Name == "mscorlib")
+                    return null;

```

Comment:
This worries my a little but without it it doesn't JIT, while with it just seems to work.